examples/asr/speech_to_text_rnnt_bpe.py

```diff
@@ -75,6 +77,13 @@ def main(cfg):
     exp_manager(trainer, cfg.get("exp_manager", None))
     asr_model = EncDecRNNTBPEModel(cfg=cfg.model, trainer=trainer)
 
+    # initialize the model with the weights of the checkpoint specified by 'load_weights_from_checkpoint' in the configs
+    # You may use this option to start the training from a pre-trained checkpoint
+    checkpoint_path = cfg.model.get("load_weights_from_checkpoint", None)
```

Comment:
Use with open_dict() and pop out this argument - it should not remain inside the config if submitted by user.